packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpIntegrationScript.tsx

```diff
@@ -1,54 +1,61 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import { RouteData } from '@sitecore-jss/sitecore-jss-nextjs';
+import { useSitecoreContext } from '@sitecore-jss/sitecore-jss-nextjs';
 import Script from 'next/script';
 import { useEffect } from 'react';
 
-declare const _boxeverq: any;
-declare const Boxever: any;
+declare const _boxeverq: {():void}[];
```

Comment:
Seems like there should be some spaces added, after `{` and before `}`. check eslint results.